jsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Are we utilizing this in boilerplate ?
If not please remove it. @jinalviranii

Reply:
Removed